Formula/basisu.rb

```diff
@@ -0,0 +1,20 @@
+class Basisu < Formula
```

Comment:
We should probably call the formula `basis_universal` to match the project name, no? We can still add a symlink to `basisu` so that users can still `brew install basisu`.

Reply:
Changed the formula name to `basis_universal` and added a `basis` alias